simulations/vip-398/arbitrumone.ts

```diff
@@ -290,5 +298,27 @@ forking(276417316, async () => {
         parseUnits("1", 7),
       );
     });
+
+    it("claim prime token", async () => {
+      await network.provider.send("hardhat_setCode", [XVS_USER, "0x"]);
+      const xvsUserSigner = await initMainnetUser(XVS_USER, parseUnits("1"));
+
+      await expect(prime.connect(xvsUserSigner).claim()).to.be.revertedWithCustomError(prime, "Unauthorized");
+
```

Comment:
Can we also verify after the user has deposited some XVS?

Reply:
done, https://github.com/VenusProtocol/vips/pull/429/commits/7c7baa1defabd4b994592d906d1d78ab88e73ea1